modern/base/Dockerfile

```diff
@@ -71,7 +71,7 @@ RUN apt-get -qq update \
     && update-alternatives --install /usr/local/bin/python python /opt/pyenv/shims/python 100 \
     && update-alternatives --install /usr/local/bin/pip pip /opt/pyenv/shims/pip 100 \
     && pip install -q --upgrade --no-cache-dir pip \
-    && pip install -q --no-cache-dir conan==${CONAN_VERSION} conan-package-tools \
+    && pip install -q --no-cache-dir conan==${CONAN_VERSION} \
```

Comment:
We needed CPT for internal tests, it was for legacy images. I think we are safe now discarding it.

Reply:
I'll keep it here. We need to get rid of it. 👍

Reply:
I remember now why we needed CPT. We are using CPT to build some packages using all possible combinations, but only in legacy images. We can re-add it to this new images, but no need to install CPT here, it can be done as entrypoint before testing.